apps/frontend/src/components/ui/charts/ChartDisplay.vue

```diff
@@ -693,6 +693,14 @@ const defaultRanges: RangeObject[] = [
     }),
     timeResolution: 40320,
   },
+  {
+    getLabel: () => "All Time",
+    getDates: (currentDate: dayjs.Dayjs) => ({
+      startDate: dayjs(0),
+      endDate: currentDate,
+    }),
+    timeResolution: 40320,
```

Comment:
this resolution should prob be dynamic based on the length of the project's history, but honestly this critique could apply to a lot of other ranges so I'll probably just adjust in the future